apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/LogLevel.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Google.Cloud.Spanner
+{
+    /// <summary>
+    /// </summary>
+    public enum LogLevel
+    {
+        /// <summary>
+        /// </summary>
+        Debug = 0,
+
+        /// <summary>
+        /// </summary>
+        Info = 1,
+
+        /// <summary>
+        /// </summary>
+        Warn = 2,
+
+        /// <summary>
+        /// </summary>
+        Error = 3,
+
+        /// <summary>
+        /// </summary>
+        None = 4
```

Comment:
It's odd to have None as an enum value of 4 - it's usually 0 when present. I understand the reason for this, but it's still surprising. A couple of options:

- Rename it to Silent or something?
- Reorder the enum from quietest to loudest?